### PR TITLE
Updating item and adding timestamps

### PR DESCRIPTION
**Pull request by keith-miller**

### CODE REVIEW

File: api/item/definition.proto [hunk 1]
```diff
@@ -19,11 +19,13 @@ message Item {
   int32 serial_number = 7;
   string amount_paid = 8;
   string currency = 9;
-  string amount_paid_base = 10;
-  string currency_base = 11;
```
Comment (axcrest) [inline api/item/definition.proto:23]: You can keep currency_base.

File: api/item/definition.proto [hunk 2]
```diff
@@ -19,11 +19,13 @@ message Item {
   int32 serial_number = 7;
   string amount_paid = 8;
```
Comment (axcrest) [inline api/item/definition.proto:20]: Remove amount_paid and currency.  Not sure if we want to try to include the new History list that is optionally available in the item object.

File: api/item/definition.proto (hunk 1 above)
Comment (keith-miller) [reply api/item/definition.proto:23]: ok

File: api/item/definition.proto (hunk 2 above)
Comment (keith-miller) [reply api/item/definition.proto:20]: ok, will do. We need to convert that transaction history into the Item state for the SDK, which we're building later.

**Review by axcrest — APPROVED** [review]

**Pull request merged**